### PR TITLE
GLUE-308 - Percy on prereleases

### PR DESCRIPTION
**Pull request by remyyounes**
https://procoretech.atlassian.net/browse/GLUE-308

### Change Log
- Re-enables Percy regression testing

### Next Steps
1. run script from circle ci

# Screenshots
screen shot 2018-01-02 at 3 46 29 pm

# Testing Requirements
- [ ] Circle CI kicks off snapshots on prerelease

### CODE REVIEW

File: packages/core-docs/scripts/visual_regression_test.rb [hunk 1]
```diff
@@ -1,23 +1,38 @@
...
 ENV['PERCY_DEBUG'] = '1'
-documented_elements = Dir["catalog/elements/*.md"]
-documented_layouts = Dir["catalog/layouts/*.md"]
 
-server = 'http://localhost:8080'
-assets_dir = File.expand_path('./build/docs')
+documented_elements = Dir["src/css/elements/*.md"]
+documented_layouts = Dir["src/css/layouts/*.md"]
+
+server = 'http://localhost:4000'
```
Comment (dannyphillips) [inline packages/core-docs/scripts/visual_regression_test.rb]: is this the port we want?

Comment (remyyounes) [reply packages/core-docs/scripts/visual_regression_test.rb]: Nope. Suggestions?

File: scripts/visual_regression_test.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  documented_elements.map { |el|
+    path = el.sub('packages/core-docs/src/css/', '').sub('.md', '').downcase
+    puts "Snapshot " + path
+    puts "==============================="
+
+    page.visit("/#/#{path}")
+    Percy::Capybara.snapshot(page, name: path)
+  }
+
```
Comment (mboperator) [inline scripts/visual_regression_test.rb]: Looks like we're not snapshotting `documented_layouts`

Comment (remyyounes) [reply scripts/visual_regression_test.rb]: addressed

**Pull request merged**